scripts/2.mutation-classifier.py

```diff
@@ -320,24 +325,75 @@ def get_threshold_metrics(y_true, y_pred):
 ])
 
 
-# In[20]:
+# In[21]:
 
 # Signs of the coefficients by model
 pd.crosstab(coef_df.feature_set, np.sign(coef_df.weight).rename('coefficient_sign'))
 
 
-# In[21]:
+# ### Top coefficients for covariates model
+
+# In[22]:
+
+coef_df.query("feature_set == 'covariates'").head(10)
+
+
+# ### Top coefficients for full model
+
+# In[23]:
+
+coef_df.query("feature_set == 'full'").head(10)
+
+
+# ### Top coefficients for individual _genes_ for full model
+
+# In[24]:
+
+pca_for_full = (
+    final_pipelines['full']
+    .named_steps['features']
+    .get_params()['expressions__pca']
+    )
+classifier_for_full = (
+    final_pipelines['full']
+    .named_steps['classify']
+    )
+num_covariates = len(covariate_df.columns)
+gene_coefficients_for_full = get_genes_coefficients(
+    pca_object=pca_for_full,
+    classifier_object=classifier_for_full,
+    expression_df=expression_df,
+    expression_genes_df=expression_genes_df,
+    num_covariates=num_covariates
```

Comment:
Let's just put `len(covariate_df.columns)` here rather than assigning it to `num_covariates`